go-controller/pkg/ovn/controller/services/lb_config.go

```diff
@@ -243,6 +281,148 @@ func buildClusterLBs(service *v1.Service, configs []lbConfig, nodeInfos []nodeIn
 	return out
 }
 
+//TODO: comment
```

Comment:
ack: TODO mentioned in the PR message

Reply:
Not at the moment.  Maybe if we start using templates for ETP/ITP local in the future but, for now, I think not.